lib/pbench/agent/toolmetadata.py

```diff
@@ -65,6 +65,7 @@ def __getInitialData(self):
         return metadata
 
     def __dataCheck(self):
+        """Check for existing/loadable data, return 1 if true and 0 if false"""
```

Comment:
Ah, that makes more sense; but maybe it should return True/False instead of integers?

We really need to be better about using description strings, in general; especially with untyped parameters, having a description can be useful to figure out what to expect.